internal/http/controller.go

```diff
@@ -178,6 +179,7 @@ func NewController(dir string) (Controller, error) {
 				client.WithURL(p.URL)
 				client.WithUsername(p.Username)
 				client.WithPassword(p.Password)
+				client.WithShortExpiration(p.ShortExpiration)
```

Comment:
We should only set this if `> 0`. Otherwise the default behavior will be to *always* fetch a token which could DDOS the rancher auth ;)